pkg/rpc/client/nep5.go

```diff
@@ -172,7 +172,11 @@ func (c *Client) CreateTxFromScript(script []byte, acc *wallet.Account, sysFee,
 		sysFee = result.GasConsumed
 	}
 
-	tx := transaction.New(c.opts.Network, script, sysFee)
+	network, err := c.GetNetwork()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create transaction: %w", err)
```

Comment:
I think we can avoid all these error modes by making a request in the `New()`. Yeah, it's not very goesque (more of RAII), but this way you'll always get a client with proper network. This behavior (implicit `getversion` request) would need to be documented though. Another (more goesque) option is to make an explicit `Init()` method that would do the request.

Reply:
We can't always perform this request in `New()` because WSClient has it's separate requests handler which is set after simple client was created. At the same time, it's easy to forget about the separate `Init()` method. That's why I decided to add `initNetwork` bool flag to `New()` in order to explicitly tell the constructor to use `getversion`. Is it good decision?